src/parser/jobs/drg/About.js

```diff
@@ -0,0 +1,25 @@
+import {Trans} from '@lingui/react'
+import React, {Fragment} from 'react'
+import {Icon, Message} from 'semantic-ui-react'
+
+import CONTRIBUTORS, {ROLES} from 'data/CONTRIBUTORS'
+import CoreAbout from 'parser/core/modules/About'
+
+export default class About extends CoreAbout {
+	description = <Fragment>
+		<Trans id="drg.about.description"><p>This analyzer aims to help you beat the sterotypes, stay off the floor, and dish out some big juicy numbers. As the DRG rotation is pretty strict, the suggestions you see will focus mostly on keeping that rotation flowing smoothly, as well as picking out issues related to your Life of the Dragon windows.</p></Trans>
```

Comment:
I'd flip the `<Trans>` and `<p>` tags around so you don't end up with the `<0>` in the i18n json.

Also, have a look at `TransMarkdown` - not necessary in this case at all, but it'll be handy for stuff like the warning below and so on.

Reply:
Weeeeeiiirrdddddd